synapse/cores/common.py

```diff
@@ -1878,6 +1881,21 @@ def _runAutoAdd(self, toadd):
                 continue
             self.formTufoByProp(form, valu)
 
+    def _reqProps(self, form, fulls):
+        if not self.enforce:
```

Comment:
So if enforce mode is off, required props are not required. (thinking aloud)

Reply:
@vertexmc this is per @invisig0th